src/app.module.ts

```diff
@@ -77,6 +78,7 @@ export class AppModule implements NestModule {
     return {
       module: AppModule,
       imports: [
+        UserRepositoryModule,
```

Comment:
This shouldn't yet be imported as you are working on the datasource. The `AppModule` should only import route layer modules.